### PR TITLE
heart rate notification email improved

### PR DESCRIPTION
**Pull request by Shurc1val**
Developed email html to look just a little cleaner:
Screenshot 2024-01-10 at 21 50 18

Also put Dawid's email in place of the rider's, firstly for the presentation (so we'll have real examples from the pipeline), and because no email is sent if but one of the addressee's is unregistered (and none of the rider's emails are registered).

### CODE REVIEW

File: pipeline/validate_heart_rate.py [hunk 1]
```diff
@@ -27,7 +27,13 @@
...
 from os import environ
 import boto3
-from botocore.config import Config
+
+
+GENDER_TITLES = {
+    'male': 'Mr',
+    'female': 'Ms',
+    'other': 'Mx'
+}
```
Comment (CharliedeanCS) [inline pipeline/validate_heart_rate.py:36]: Nice use of a constant

File: pipeline/validate_heart_rate.py [hunk 2]
```diff
@@ -115,28 +121,40 @@ def send_email(rider_details: dict, extreme_hr_counts: list[int]) -> None:
...
-    first_name = rider_details.get("first_name")
     last_name = rider_details.get("last_name")
+    gender = rider_details.get('gender')
 
     ses_client.send_email(
         Destination={
             "ToAddresses": [
-                rider_email,
-                "trainee.hugh.morris@sigmalabs.co.uk"
+                # rider_email,  (removed because they are currently unverified through AWS)
```
Comment (CharliedeanCS) [inline pipeline/validate_heart_rate.py:130]: Why not just set rider_email above to dawid's?

Comment (Shurc1val) [reply pipeline/validate_heart_rate.py:130]: Because this isn't how we would ideally have the implementation – wanted to make this very clear.

**Review by CharliedeanCS — APPROVED** [review]
New email structure to the heart rate alert and a test email has worked correctly.

**Pull request merged**